docs/developer/process/integration-release.md

```diff
@@ -225,6 +229,44 @@ securesystemslib.exceptions.FormatError: '[none]' did not match 'pattern /[a-fA-
         deleted tags, so any subsequent manual trigger tags will need to increment the version number.
 
     1. Delete the branch and tag, locally and on GitHub.
+    
+- If the [build pipeline](../meta/cd.md) failed due to another feature PR conflicting with and being merged after the release PR is opened, the merged release PR will not have updated and signed the feature PR's files.
```

Comment:
What's the difference between this case and the one above? I know there's a difference, but it's unclear which one to use when.

Reply:
The first case is more common, where it's just a missing signature like PR was pushed but yubikey signing failed so only the changelog and integration versions were bumped

Reply:
K, then maybe make that clear? 🙂 First, try this. If not, then try that...